tarot-time/TarotTimeClient
# About page

Added about page, configured to be accessible via footer regardless of sign-in status. 

Steps to test: git fetch --all 
git checkout about-page

Launch server and client

Verify Tarot Time footer appears with link to About page on all pages
Verify About page loads correctly

Copy is still first draft, probably manyyyy updates to come!
